manifests/rbac/controller.yaml

```diff
@@ -69,6 +69,11 @@ rules:
   - update
   - patch
   - delete
+# required for flow control
+- nonResourceURLs:
+    - /livez/ping
+  verbs:
+    - head
```

Comment:
Can you please use the same indentation

Reply:
:facepalm: Copy/paste from local verification was linted